scripts/backup/pg_restore.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
```

Comment:
I wasted some time to understand how both scripts work and how to execute them . I think we need to stay doc string there or update readme. Let's leave puzzle for that?